keyboards/rart/rartlite/readme.md

```diff
@@ -0,0 +1,14 @@
+# [RARTLITE](https://github.com/alabahuy/RART/tree/master/RARTLITE)
+
+![Top](https://user-images.githubusercontent.com/30220306/107148534-4212e200-6986-11eb-824d-dd7fbe10d333.png)
+40% Layout mechanical keyboard with support left and right numpad. based on IMKG (Indonesia Mechanical Keyboard Group)
+
+* Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)
+* Hardware Supported: RARTLITE PCB, Promicro, Mini USB
+* Hardware Availability: Private GB
+
+Make example for this keyboard (after setting up your build environment):
+
+    make rart/rartlite:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Could you include instructions on how to enter the bootloader?

Reply:
Pro micro doesn't need to add another bootloader right?

Reply:
Nope.  But what I mean is how to enter the bootloader so you can flash the firmware.  Eg, does it support bootmagic, is there a dedicated hardware reset button, etc.

Reply:
ok, done :)